modules/svg/labels.js

```diff
@@ -57,6 +57,17 @@ export function svgLabels(projection, context) {
         ['point', 'shop', '*', 10],
         ['point', 'tourism', '*', 10],
         ['point', 'camp_site', '*', 10],
+        ['*', 'alt_name', '*', 12],
+        ['*', 'official_name', '*', 12],
+        ['*', 'loc_name', '*', 12],
+        ['*', 'loc_ref', '*', 12],
+        ['*', 'unsigned_ref', '*', 12],
+        ['*', 'seamark:name', '*', 12],
+        ['*', 'sector:name', '*', 12],
+        ['*', 'addr:housename', '*', 12],
+        ['*', 'lock_name', '*', 12],
+        ['*', 'distance', '*', 12],
+        ['*', 'railway:position', '*', 12],
```

Comment:
Should this be limited to points? [The `railway:position` documentation](https://wiki.openstreetmap.org/wiki/Key:railway:position) says this key can be used on any kind of feature, but it seems to be [extremely rare](https://taginfo.openstreetmap.org/keys/railway%3Aposition#overview) on ways and relations, and all of the presets that use the Milestone Position field are for nodes only.

Reply:
sure, we could limit it to points, but also keep in mind that most of the geometry rules in this array have no effect, because of these 3 wildcard entries:

https://github.com/openstreetmap/iD/blob/9cf0a20592ce8792aa14f2faeb6b30f122101434/modules/svg/labels.js#L74-L76